docs/zh-cn/how_keyboards_work.md

```diff
@@ -0,0 +1,80 @@
+# 计算机是如何注册并解释键的
+
+在本文中，你将会学习键盘通过USB工作的概念，
+你将会更好的理解直接改变你的固件可以得到什么。
+
+
+## 示意图
+
+当你敲击某个键时，将会发生以下一系列事情
+如下所示：
+
+``` text
++------+         +-----+       +-----+      +-------+     +----------+
+| 用户 |-------->| 键  |------>| 固件 |----->| USB线 |---->| 操作系统 |
++------+         +-----+       +-----+      +-------+     +----------+
+```
+
+示意图简化的阐释了发生的事情，接下来的章节中将会详细解释。
+
+
+## 1. 按下某个键
+
+每当你按下某个键，你键盘的固件就记录了这个事件。<!--TODO:翻译成记录的这个词是：register，也可能翻译成"注册"，暂定"记录"，有更好的请改正>
```

Comment:
This HTML comment needs to be closed properly, so it doesn't get rendered to the page.

![image](https://user-images.githubusercontent.com/18669334/75934015-3a6ec300-5e30-11ea-96c5-ac3a66d4d9e3.png)

```suggestion
每当你按下某个键，你键盘的固件就记录了这个事件。<!--TODO:翻译成记录的这个词是：register，也可能翻译成"注册"，暂定"记录"，有更好的请改正-->
```